pkg/injector/consts/consts.go

```diff
@@ -37,6 +37,7 @@ const (
 	UserContainerAppProtocolName   = "APP_PROTOCOL"                         // Name of the variable exposed to the app containing the app protocol.
 	UserContainerDaprHTTPPortName  = "DAPR_HTTP_PORT"                       // Name of the variable exposed to the app containing the Dapr HTTP port.
 	UserContainerDaprGRPCPortName  = "DAPR_GRPC_PORT"                       // Name of the variable exposed to the app containing the Dapr gRPC port.
+	DaprContainerHostIP            = "DAPR_HOST_IP"                         // Name of the variable injected in the daprd container containing the pod's IP
```

Comment:
I'm very late to the party :) Why it was named `DAPR_HOST_IP` when it's really DAPR_POD_IP? I was looking for a way to get HOST_IP into the daprd sidecar to configure `OTEL_EXPORTER_OTLP_ENDPOINT` with `HOST_IP` and  this name makes it confusing. Even the comment says that it's a pod's IP.